notebooks/initial_EDA.ipynb

```diff
@@ -0,0 +1,847 @@
+{
```

Comment:
<p>The notebook LGTM. Just one comment:</p><p> </p><p>I found it a little hard to understand the process of finding correlation. Would it help to add a small reasoning or steps here, maybe something like</p><p>1) Encoding: Representing jobs as a vector of tests</p><p>2) Removing tests that are not frequent</p><p>3) Finding correlation matrix (which I guess would be interpreted as highly correlated tests fail together?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Thanks Shrey, I'll try to make if clearer.